### PR TITLE
Added In Plain English

### PR DESCRIPTION
**Pull request by Ashimi0x**
Hello @malgamves @karllhughes 

I discovered "In Plain English" recently accepts writers so I included it here as I think it would be needful

### CODE REVIEW

**Comment by john-walter-munene** [comment]
Is is a paid community program??

**Comment by kimkimani** [comment]
@Ashimi0x Could you please provide more details on how they handle payments? I tried asking them before submitting my first topic, and I got the following response:

Screenshot 2023-10-04 094500
